ui/components/table.js

```diff
@@ -96,12 +110,12 @@ export default function Table({
         <thead className='border-b border-gray-200/75 bg-zinc-50/50 text-xs text-gray-500'>
           {table.getHeaderGroups().map(headerGroup => (
             <tr key={headerGroup.id}>
-              {allowDelete && data?.length > 0 && (
+              {allowDelete && data?.length > 1 && (
```

Comment:
I'd like the checkbox remove option even if there's a single item. The user doesn't need to change their what their doing just because the contents of the table changed. Maybe even have the checkbox available when there's no items but grey out the `Remove` button?